resources/default_pipeline_environment.yml

```diff
@@ -337,13 +338,17 @@ steps:
     logSuccessfulMavenTransfers: false
   mtaBuild:
     buildTarget: 'NEO'
-    mtaBuildTool: classic
+#    mtaBuildTool: classic
+    mtaBuildTool: 'cloudMbt'
     platform: 'cf'
     mtaJarLocation: '/opt/sap/mta/lib/mta.jar'
     classic:
       dockerImage: 'ppiper/mta-archive-builder'
     cloudMbt:
-      dockerImage: 'devxci/mbtci:1.0.4'
+#      dockerImage: 'devxci/mbtci:1.0.4'
+      dockerImage: 'alunde/mbtci:latest'
```

Comment:
we should stay with the official image and not switch to a custom one or will the `alunde/mbtci` be the official one?

Reply:
This change isn't needed.  I was having trouble figuring out why my config.yaml settings weren't getting updated and ended up changing the default.